app/code/Magento/InventoryInStorePickup/Test/Integration/PickupLocation/MapperTest.php

```diff
@@ -85,86 +88,25 @@ public function testWrongMappingForPickupLocation()
     }
 
     /**
-     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/source.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryInStorePickup/Test/_files/pickup_location.php
      */
     public function testMapPickupLocation()
```

Comment:
What about case: 
`Source.ExtensionAttributes.Field1 -> PickupLocation.ExtensionAttributes.Field2` ?

Reply:
Why did you not cover it before?)

Okay, I`ll check

Reply:
@swnsma , I afraid it is out of scope. Feel free to do it by yourself.